### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by Mbrick2** [Chiroyce1/Chiroyce1#11] [General]
https://github.com/Mbrick2/blog is not deploying

**Comment by Chiroyce1** [comment]
Will check and let you know.

**Comment by Chiroyce1** [comment]
Looks like you have an error in the yml file? https://github.com/Mbrick2/blog/runs/6022536939?check_suite_focus=true#step:4:33

**Comment by Chiroyce1** [comment]
Can you try copying my yml file from [Chiroyce1/blog](https://github.com/Chiroyce1/blog/blob/master/_config.yml) and then see if it works? After that you can modify it as you wish.

**Reply by Mbrick2** [reply]
Thanks
